core/includes/updater.inc

```diff
@@ -181,7 +181,8 @@ abstract class Updater implements BackdropUpdaterInterface {
    * @throws UpdaterException
    */
   public static function getProjectType($directory) {
-    if ($directory == BACKDROP_ROOT) {
+    $is_backdrop = explode('/', $directory)[3];
```

Comment:
this syntax won't work, can you please update to match Backdrop coding standards?